### DISCUSSION TITLE
Typo in the blog post

### DISCUSSION THREAD

**Discussion by zikato** [yonileibowitz/yonileibowitz.github.io#15] [General]
Hi Yoni, great blog!

I've noticed a typo in post https://y0nil.github.io/kusto.blog/blog-posts/workload-groups.html

the `.alter cluster policy request_classification '{"IsEnabled":true} <|` example is missing an apostrophe.

It should be  `.alter cluster policy request_classification '{"IsEnabled":true}' <|`

**Comment by y0nil** [comment]
thank you @zikato for letting me know. i've submitted a fix for that: https://github.com/y0nil/kusto.blog/pull/16
